js/jquery.validation.js

```diff
@@ -85,40 +80,83 @@
             
             // Handler function for submit event.            
             $form.on('submit.validate', function(event) {
-                var $form = $(this);
 
-                if ( typeof FormData === "function" ) {
-                    // AJAX file upload supported
+                // Default url used for validation service.
+                if ( typeof widget.options.url === 'undefined' ) {
+                    var url = $form.attr('action');
+                }
+
+                if ( typeof FormData === "function"
```

Comment:
I've tested the first path by validating a form with the "PUT" method in chrome. I've tested the second by doing the same in IE (which does not support "get" or "set"). I don't have a browser installed with no FormData support - would IE in emulation mode be sufficient?